conftest.py

```diff
@@ -0,0 +1,34 @@
+import pytest
```

Comment:
Should this be located at `manubot/conftest.py` rather than in the root?

Reply:
Indeed, `conftest.py` should be where `__init__.py` is.

Reply:
https://github.com/manubot/manubot/pull/151/commits/a64ba13533181afddc53737f7c469935e44d0ec9